.dockerignore

```diff
@@ -1,5 +1,3 @@
-.git
-.git/**
```

Comment:
if ignore .git will have error when build gogs

```
fatal: not a git repository (or any of the parent directories): .git
go install "-v" -ldflags '-X "github.com/gogs/gogs/pkg/setting.BuildTime=2018-07-18 01:38:48 UTC" -X "github.com/gogs/gogs/pkg/setting.BuildGitHash="' -tags 'sqlite cert pam'

```

Reply:
As long as the new Dockerfile build steps do not increate final image size because of coping the whole Git data, I think it is fine to include.